### DISCUSSION TITLE
RKE2 selinux config change not persisting

### DISCUSSION THREAD

**Discussion by clp01** [rancher/rke2#4779] [Q&A] [answered]
Hi All,

I have a Rancher deployed RKE2 cluster (`v1.25.12+rke2r1`) and have installed `rke2-selinux`. I added `"selinux": true` to `/etc/rancher/rke2/config.yaml.d/50-rancher.yaml` and restarted the rke2 service on each node. I then set `SELINUX=enforcing` in `/etc/selinux/config` and rebooted the server, however I discovered that the reboot caused the rke2 config change to be reverted.

What's the correct way of applying this change? I couldn't see an option to enable selinux support when deploying the cluster in Rancher.

Thanks!

**Comment by Martin-Weiss** [comment, marked as answer]
In case you have created the cluster with Rancher you have to change the configuration with Rancher as well (edit cluster).

**Reply by brandond** [reply]
Or if you must, just create another yaml file in that directory and put your changes in there. Anything other than editing the file that is managed by Rancher... in case that was not made clear by its name and the fact that edits to it do not persist.

**Reply by Martin-Weiss** [reply]
@brandond  - does RKE2 fully support changing the setting for selinux on an already deployed cluster? (do we have a list of parameters that are allowed and supported to be changed after creating the RKE2 cluster initially?)

**Reply by brandond** [reply]
It's all local to the node, there is nothing changed at the cluster level by enabling it. If you install the selinux rpms and restore the contexts, it should work fine.
